### ISSUE TITLE
Laggy moment in game (with complete CDDA folder INC) reason below

### ISSUE THREAD

**Issue by stk2008** [CleverRaven/Cataclysm-DDA#25043]
[**Describe ] the bug**  
I am on day 11 or so and I was going to make some more jerky using my charcoal smoker which is just out side to the West.

I went out and starting CRAFTING jerky and it was taking a very long time...As if the game was lagging.

**To Reproduce**  
skip time where you spawn in (1800 heart beats) notice lag.
pick up the meat near the door to your LEFT.
Go out side to the door you will see a Charcoal smoker.
Craft some Jerky game laggs and takes a long time to complete task.
Now use wish menu (bound to DEL key).
move over to the EAST one sqaure or two and plant your self down.
Skip time (1800 heart beats) notice game is fine now.
go back to charcoal smoker redo jerky task note game is fine now.

There is hardly any monsters out side.
There is a spider splashing its damn webs every where.
There is one to the East and West I think....though the one to the East I aint seen now for a while

**Expected behavior**  
To not start lagging all of a sudden.

**Screenshots**  
https://i.imgur.com/ORdQVR0.png

https://i.imgur.com/fXFNJvz.jpg

**Versions and configuration(please complete the following information):**
 - OS: windows 10 64bit
 - Game Version 64 bit tiles version 7700
 - Graphics version Tiles
 - Mods loaded 54 odd (look in mods folder)

**Additional context**  
Thank you and so sorry for the large file size :(
Reason for whole folder it inc saves
and I have a lot of mods

This is the whole CDDA folder I am so so sorry

https://drive.google.com/file/d/1ZwLcxe_6BJ9ti4BVdvI9C3gkD0r0YDUB/view?usp=sharing

**Comment by Treah** [comment]
I have noticed this as well in random locations in the game world. Not sure what was causing it or even how to reproduce it. I've noticed it for quite some time tho ( least 3 monthes ).

**Comment by stk2008** [comment]
Yeah well hopefuly now I have made this reproducable and INC the save etc they can figure out whats causing it :) lets hope ah.

**Comment by kevingranade** [comment]
An annoymous report of a similar slowdown while crafting resolved the lag by recruiting a nearby NPC, indicating that wild NPC AI may be the cause.
Profiling by @ZhilkinSerg indicates that the time is being spent in mao::route()

**Comment by ZhilkinSerg** [comment]
Savegame attached.

[Fort Lauderdale.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/2384357/Fort.Lauderdale.zip)

![image](https://user-images.githubusercontent.com/16213433/45566210-ac2f0780-b85e-11e8-9bc4-39a43eedac9e.png)

**Comment by kevingranade** [comment]
This might be related: https://discourse.cataclysmdda.org/t/mi-gos-and-minefields/16614?u=kevin.granade

Tl;dr mi-go observed to get "stuck" while walking through a minefield immediately before stepping on a mine.  Speculation is that it is pathing through the mine, but when it comes time to take that step it checks for danger and cancels the move, but then keeps re-pathing through it.

**Comment by TinkerWorX** [comment]
In the following save, there's an NPC nearby(west-northwest) and the game is lagging. If you go closer to him, it'll stop lagging. He doesn't appear to even be blocked.

[save.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/2424592/save.zip)

**Comment by ZhilkinSerg** [comment]
It seems to be resolved now.
